analysis_driver/report_generation/report_crawlers.py

```diff
@@ -64,10 +65,13 @@ def _populate_barcode_info_from_sample_sheet(self, samplesheet):
             for sample_id_obj in proj_obj.sample_ids.values():
                 for sample in sample_id_obj.samples:
                     for lane in sample.lane.split('+'):
-                        run_element_id = '%s_%s_%s' % (self.run_id, lane, sample.barcode)
```

Comment:
We should use a two element id when barcode is not present.
We should also remove the ELEMENT_BARCODE entry from the dict